src/get-source.ts

```diff
@@ -58,8 +58,16 @@ function getParamsInterface(
 ): string {
   const props: string[] = entries.map(([key, value]) => {
     const type = getPropTypes(value);
+    // construct dependency list per parameter
+    let dependencyTsdocLines: string[] = [];
+    if ('depends' in value) {
+      const dependencies: string[][] = Array.from(value.depends.map((depend: string) => [depend.includes("=") ? depend.split("=")[0] : depend, depend]))
```

Comment:
Some dependencies are expressed with an explicit value, e.g. `[ 'trim', 'trim=color' ]`. This line handles the latter as the `=` can mess up the way the tsdocs get rendered in the IDE.